### ISSUE TITLE
Yolov8Inference when use retina_masks=True is slow

### ISSUE THREAD

**Issue by andreazuna89** [IASRobolab/ai_utils#14]
Yolov8 gives as output the masks which have the reduced size of the processed input image (480x640). If we want the masks with the original image dimensions we need to put retina_masks=True in the input parameters but this is slowing down the speed of the inference.

**Comment by andreazuna89** [comment]
We found a block of code which was slowing down the speed when we were using retina_masks=True. We replaced this block of code in Yolov8Inference:
```
boxes = results.boxes.xyxy.tolist()
masks = results.masks.masks.tolist()
classes = results.boxes.cls.tolist()
scores = results.boxes.conf.tolist()
```

with:
```
boxes = results.boxes.xyxy.detach().cpu().numpy()  
masks = results.masks.masks.detach().cpu().numpy()
classes = results.boxes.cls.detach().cpu().numpy()
scores = results.boxes.conf.detach().cpu().numpy()
```
